package.json

```diff
@@ -30,6 +30,7 @@
 		"build": "lerna run build",
 		"build:browsertest": "lerna run build:browsertest",
 		"cover": "lerna run cover",
```

Comment:
Since we're combining the two scripts now, we should have a root `clean` script which just runs `lerna run clean`.